Add missing Rust update steps (#712)
ubuntu/ubuntu-project-docs: docs/maintainers/niche-package-maintenance/rustc/common/obtaining-upload-info.md

$ lintian > <path_to_saved_lintian_output_file>
```

> If you are updating the toolchain to a new version (necessitating a new package upload), an {term}`Archive Admin` will need to approve the new package. You can make their job easier and speed up your package upload by generating a diff of the `debian/` directory and including it as an attachment, allowing the Archive Admin to easily review the changes:

Reviewer: ```suggestion
For updating the toolchain to a new version (necessitating a new package upload), an {term}`Archive Admin` needs to approve the new package. To make their job easier and speed up the package upload by generating a diff of the `debian/` directory and including it as an attachment, allowing the Archive Admin to easily review the changes:
```

- A list of any notable packaging changes (_not_ upstream changes). These are changes _you_ made to the package, such as the removal or addition of patches. Nothing routine (such as patch refreshes or vendored dependency pruning) should be added here.
- The output of `lintian` you just got
- The links to the build logs of all PPA autopkgtests you just got
- If this is a Rust update, an attachment of the diff of the `debian/` directory between the old and new Rust versions

> You can see a real example of such an upload request [here](https://bugs.launchpad.net/ubuntu/+bug/2156635/comments/1).

Reviewer: ```suggestion
- For Rust updates, an attachment of the diff of the `debian/` directory between the old and new Rust versions

See a real example of such an upload request at {lpbug}`2156635`.
```

Author: Apropos the Launchpad URL, I'd like to link to the comment specifically rather than the bug as a whole just so the reader can focus on the upload info itself rather than the surrounding bug and discussion. In this case, can we keep the unformatted URL?
